player/webpack.plugin.config.js

```diff
@@ -21,8 +21,12 @@ const UglifyJsPlugin = require("uglifyjs-webpack-plugin")
 const WebpackOnBuildPlugin = require("on-build-webpack")
 const APP_CONFIG = require("./build.config.js")
 
+const BUILD_NUMBER = process.env.build_number;
+const PLAYER_VER = process.env.player_version_number;
+const VERSION = PLAYER_VER + '.' + BUILD_NUMBER;
+
 const PLUGINS_BASE_PATH = "./public/coreplugins/" // Plugins base path
-const PACKAGE_JS_FILE_NAME = "coreplugins.js" // Packaged all plugins js file name
+const PACKAGE_JS_FILE_NAME = "coreplugins." + VERSION + ".js" // Packaged all plugins js file name
```

Comment:
Have you tested the preview build in editor after making these changes? Is corePlugins.js file is loading after appending the version to it?